<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QmitkImageAndTransformSenderWidget</class>
 <widget class="QWidget" name="QmitkImageAndTransformSenderWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>605</width>
    <height>561</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_5">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="ctkCollapsibleGroupBox" name="m_GroupBox">
       <property name="title">
        <string>OpenIGTLink</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="m_ImageLabel">
            <property name="toolTip">
             <string>Select one set of target fiducials</string>
            </property>
            <property name="text">
             <string>image</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QmitkDataStorageComboBox" name="m_ImageCombo"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="m_TransformLabel">
            <property name="toolTip">
             <string>Select one set of moving fiducials, of same size as target fiducials</string>
            </property>
            <property name="text">
             <string>transform</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QmitkDataStorageComboBox" name="m_TransformCombo"/>
          </item>
          <item row="7" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QSpinBox" name="m_ImagePortSpinBox">
              <property name="maximum">
               <number>20000</number>
              </property>
              <property name="value">
               <number>18945</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="m_ImageStartServer">
              <property name="text">
               <string>start server</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="9" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="ctkPathLineEdit" name="m_DirectoryPath" native="true"/>
            </item>
            <item>
             <widget class="QPushButton" name="m_StartRecording">
              <property name="text">
               <string>start recording</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="3" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QSpinBox" name="m_TransformPortSpinBox">
              <property name="maximum">
               <number>20000</number>
              </property>
              <property name="value">
               <number>18944</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="m_TransformStartServer">
              <property name="text">
               <string>start server</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="m_TransformPortLabel">
            <property name="text">
             <string>transform port</string>
            </property>
           </widget>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="m_ImagePortLabel">
            <property name="text">
             <string>image port</string>
            </property>
           </widget>
          </item>
          <item row="9" column="0">
           <widget class="QLabel" name="m_DirectoryLabel">
            <property name="text">
             <string>directory</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QmitkDataStorageComboBox</class>
   <extends>QComboBox</extends>
   <header>QmitkDataStorageComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>ctkCollapsibleGroupBox</class>
   <extends>QGroupBox</extends>
   <header>ctkCollapsibleGroupBox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkPathLineEdit</class>
   <extends>QWidget</extends>
   <header>ctkPathLineEdit.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
